static/js/humhub/humhub.ui.form.elements.js

```diff
@@ -150,14 +150,54 @@ humhub.module('ui.form.elements', function (module, require, $) {
         evt.$trigger.parent().hide();
     };
 
+    var object = require('util').object;
+    var Widget = require('ui.widget').Widget;
 
+    var FormFieldsCollapsible = function (node, options) {
```

Comment:
Just for your info: This block and line 160 can be replaced with `var FormFieldsCollapsible = Widget.extend();`

Reply:
Ok, I followed your example here: https://github.com/humhub/humhub/blob/master/static/js/humhub/humhub.ui.form.js#L3-L9